tool/webqa-normalize-data.js

```diff
@@ -498,8 +499,10 @@ module.exports = {
             addHelp: true,
             description: "Normalize schema.org JSON+LD files to match their ThingTalk representation."
         });
-        parser.addArgument(['-o', '--output'], {
-            required: true,
+        parser.addArgument('--data-output', {
+            type: fs.createWriteStream
+        });
+        parser.addArgument('--meta-output', {
```

Comment:
It's somewhat unfortunate to change the command-line in a stable release.
It is also annoying to have a command line tool with two outputs, mostly because make really doesn't like that.

Also, is there a reason why these are not required outputs?

Reply:
I don't want to create a separate script to do the job that has been done... any suggestions? 

they are not required, so i can choose to output only one in the Makefile :)